app/components/file_upload_preview/file_upload_preview.js

```diff
@@ -107,7 +129,7 @@ export default class FileUploadPreview extends PureComponent {
 
         return (
             <KeyboardLayout>
-                <View style={[style.container]}>
+                <View style={[style.container, {height: this.state.deviceHeight}]}>
```

Comment:
Could this just have the height set to 100%?

Reply:
100% height won't work for absolute containers